torch_frame/data/multi_embedding_tensor.py

```diff
@@ -201,6 +208,19 @@ def _single_index_select(
             )
         assert False, "Should not reach here."
 
+    def _fill_col(
+        self,
+        index: int,
+        fill_value: Union[float, int],
+    ) -> None:
+        value_index = slice(self.offset[index], self.offset[index + 1])
+        values = self.values[:, value_index]
+        if self.values.is_floating_point():
+            values[torch.isnan(values)] = fill_value
+        else:
+            values[values == -1] = fill_value
+        self.values[:, value_index] = values
```

Comment:
Do we need this? Looks like everything before is zero-copy anyway.

Reply:
Thanks. What you say make sense!